packages/cli/src/commands/composite/create.ts

```diff
@@ -23,12 +24,22 @@ export default class CreateComposite extends Command<Flags, { schemaFilePath: st
       char: 'o',
       description: 'path to the file where the composite representation should be saved',
     }),
+    deploy: Flags.boolean({
+      char: 'd',
+      description:
+        'Deploy the composite to the ceramic node, which will start indexing on the composite',
+      default: true,
```

Comment:
Also, what does `composite:from-model` do?  That one seems even more like it definitely shouldn't auto-deploy since you're likely to want to add indices to it before deploying.  And if that one doesn't auto-deploy, then I feel like this shouldn't either just for consistency sake if nothing else

Reply:
I was doing this just to maintain the existing behavior, so this change won't suddenly cause things to "stop working" for users that aren't calling deploy. 

I'm not entirely sure which way is better either.

Reply:
okay, I guess preserving existing behavior makes sense